public/app/features/plugins/sql/components/query-editor-raw/QueryEditorRaw.tsx

```diff
@@ -39,7 +38,7 @@ export function QueryEditorRaw({ children, onChange, query, width, height, compl
       height={height}
       query={query.rawSql!}
       onChange={onRawQueryChange}
-      language={{ id: 'sql', completionProvider, formatter: formatSQL }}
```

Comment:
This change was not necessary for this PR but it can help in the future to be able to get language definition for mysql and postgres in the future.